easybuild/easyconfigs/t/Tensorflow/Tensorflow-1.4.1-goolf-1.7.20-Python-3.5.2.eb

```diff
@@ -0,0 +1,73 @@
+easyblock = 'Bundle'
+
+name = 'Tensorflow'
```

Comment:
@pescobar Please rename to `TensorFlow`, see #5495.

Also missing deps & several style issues to fix...